cnf-certification-test/accesscontrol/suite.go

```diff
@@ -209,8 +209,40 @@ var _ = ginkgo.Describe(common.AccessControlTestKey, func() {
 		testhelper.SkipIfEmptyAny(ginkgo.Skip, env.Pods)
 		test1337UIDs(&env)
 	})
+
+	testID, tags = identifiers.GetGinkgoTestIDAndLabels(identifiers.TestAutomountToken)
+	ginkgo.It(testID, ginkgo.Label(tags...), func() {
+		testhelper.SkipIfEmptyAny(ginkgo.Skip, env.Pods)
+		testProjectedVolumeServiceAccount(&env)
+	})
 })
 
+func testProjectedVolumeServiceAccount(env *provider.TestEnvironment) {
+	ginkgo.By("Testing pods to ensure they are not using projected volumes for service account access")
+	var badPods []string
+	index := 0
+	for _, put := range env.Pods {
+		for i := range put.Spec.Volumes {
+			if len(put.Spec.Volumes) == 0 || put.Spec.Volumes[i].Projected == nil {
+				continue
+			}
+			if put.Spec.Volumes[i].Projected.Sources == nil {
+				tnf.ClaimFilePrintf("Pod: %s/%s do not use projected volumes and service account tokens %s", put.Namespace, put.Name, put.Spec.Volumes[i].Name)
```

Comment:
tnf.ClaimFilePrintf("Pod: %s/%s do not use projected volumes and service account tokens %s", put, put.Name, put.Spec.Volumes[i].Name)
=>
tnf.ClaimFilePrintf("%s, volume=%s does not use projected volumes", put, put.Spec.Volumes[i].Name)
or create a new pod stringer that includes volume